src/actions/index.ts

```diff
@@ -251,8 +268,7 @@ const _refreshOperations: ThunkAction<
   AppState,
   void
 > = async dispatch => {
-  // TODO: API calls should be wrapped in try/catch blocks; that can be encapsulated
-  // into a helper.
+  // TODO: transition to API call helper
```

Comment:
I don't want to make this revision any longer than it is, but let's make sure this gets done next since you've already created the GetOperations API call types.